common/internal/queues/tx_priority_queue_test.go

```diff
@@ -0,0 +1,252 @@
+package queues_test
+
+import (
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/v2/common/internal/queues"
+	txmgrtypes "github.com/smartcontractkit/chainlink/v2/common/txmgr/types"
+	evmgas "github.com/smartcontractkit/chainlink/v2/core/chains/evm/gas"
+	evmtypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/types"
+)
+
+func TestTxPriorityQueue(t *testing.T) {
```

Comment:
It'd be good to have more corner case testing for the priority queue. Also are there no tests for the priority heap?